vermouth/tests/test_ff_files.py

```diff
@@ -36,6 +36,22 @@ def test_moleculetype():
         assert 'GLY' in ff.blocks
         assert ff.blocks['GLY'].nrexcl == 3
 
+    @staticmethod
+    def test_multiple_moleculetype():
+        lines = """
+            [ moleculetype ]
+            GLY  3
+            
+            [ moleculetype ]
+            VAL 2
+            """
+        lines = textwrap.dedent(lines).splitlines()
+        ff = vermouth.forcefield.ForceField(name='test_ff')
+        vermouth.ffinput.read_ff(lines, ff)
```

Comment:
This (code) layout does mean that FFs cannot be named using an `info.ff` file. They instead will always get their name from the folder. Again, not bad, but it should be a deliberate choice.